engine/src/AnimatedSpritesUpdater.cpp

```diff
@@ -0,0 +1,25 @@
+//
+// Created by tdancois on 8/4/16.
+//
+
+#include "AnimatedSpritesUpdater.h"
+
+AnimatedSpritesUpdater *AnimatedSpritesUpdater::instance;
+
+AnimatedSpritesUpdater::AnimatedSpritesUpdater() {}
+
+AnimatedSpritesUpdater *AnimatedSpritesUpdater::getInstance() {
```

Comment:
What I mean is that with this version you have to manage the memory yourself. With mine, you let the computer do the crappy job.